adapter/src/hermes/adapter/constants.h

```diff
@@ -28,4 +28,29 @@ const char* kHermesConf = "HERMES_CONF";
  * from I/O. This is needed as we should not intercept these files.
  */
 const char* kHermesExtension = ".hermes";
+/**
+ * kHermesBufferMode env variable is used to define the mode in which Hermes'
```

Comment:
Should this be "ADAPTER_MODE env variable?"

Reply:
FIXED.